packages/@aws-cdk/aws-codebuild/test/codebuild.test.ts

```diff
@@ -1028,6 +1028,93 @@ describe('secondary sources', () => {
   });
 });
 
+describe('sources with customised build status configuration', () => {
+    test('github', () => {
+        const context = "My custom CodeBuild worker!"
+        const stack = new cdk.Stack();
+        const source = codebuild.Source.gitHub(
+            { owner: 'awslabs', repo: 'aws-cdk', buildStatusContext: context }
+        );
```

Comment:
Generally, this is not how we indent these in the project -probably pays to be consistent:

```suggestion
        const source = codebuild.Source.gitHub({
          owner: 'awslabs',
          repo: 'aws-cdk',
          buildStatusContext: context,
        });
```